src/Resources.resx

```diff
@@ -0,0 +1,890 @@
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+	<data name="BI0000" xml:space="preserve">
+		<value>BI0000: Unexpected error - Please fill a bug report at https://github.com/xamarin/xamarin-macios/issues/new</value>
+		<comment>
+		</comment>
+	</data>
+
+	<data name="BI0001" xml:space="preserve">
+		<value>The .NET runtime could not load the {0} type. Message: {1}</value>
+		<comment>
+		</comment>
+	</data>
+
+	<data name="BI0002" xml:space="preserve">
+		<value>Could not compile the API bindings.</value>
+		<comment>
+		</comment>
+	</data>
+
+	<data name="BI0026" xml:space="preserve">
+		<value>Could not parse the command line argument '{0}': {1}</value>
+		<comment>
+		</comment>
+	</data>
+
+	<data name="BI0068" xml:space="preserve">
+		<value>Invalid value for target framework: {0}.</value>
+		<comment>
+		</comment>
+	</data>
+
+	<data name="BI0070" xml:space="preserve">
+		<value>
+				Invalid target framework: {0}. Valid target frameworks are: {1}.
```

Comment:
are the extra spaces inside the `<value/>` significant ?

Reply:
No I don't think so. I will remove them.

Reply:
Not going to remove the spaces because they look fine in vsmac. 

![Screen Shot 2019-12-03 at 10 45 56 AM](https://user-images.githubusercontent.com/54864665/70066202-4a210600-15ba-11ea-879f-32fa424a3d36.png)

Reply:
as long as they don't affect the output (on terminal or the IDE) then it's fine by me :)